test/cli/test_cli_functions.sh

```diff
@@ -668,3 +668,37 @@ function check_backingstore {
     echo_time "💬  Deleting bucket testbucket"
     test_noobaa bucket delete "testbucket"
 }
+
+function check_dbdump {
+    # Generate db dump
+    echo_time "💬  Generaiting db dump"
+    test_noobaa db-dump --dir /tmp
+
+    # Check whether dump was created
+    dump=`ls -l /tmp | grep noobaa_db_dump | awk '{ print $9 }'`
+    if [ -z "$dump" ]
```

Comment:
you can use `-f` 
```suggestion
    local dump="/tmp/noobaa_db_dump"
    if [ ! -f ${dump} ]
```

Reply:
I've converted to -f, even though it makes little difference.
All comments related to the value of dump however are incorrect IMO, as the file has timestamp suffix (similar to the diagnose tar), so it is required to use grep and awk to locate it